### ISSUE TITLE
Preklad ZZ do HU

### ISSUE THREAD

**Issue by milant007** [zauzoo/doc#1]
Pozadavek z Madarska :-) :
"Dále jeden madar umi anglicky ale zbytek ne, takze bych potreboval nejakej excel aby si to mohli prelozit z Anglictiny do madarstiny."

**Comment by jzelenka** [comment]
to je pozadavek na lokalizaci aplikace?
O cemz se trvdilo ze to u Volva nebude?

**Comment by milant007** [comment]
ano, presne tak. ja jim to taky psal ze chteli ZZ v anglictine. Ale na druhou stranu se ptam i tebe kdyby na tom najednou trvali...

**Comment by jzelenka** [comment]
Jo je to v planu, ale je to vetsi vec....
